packages/helpers/src/product/getProductOptions.ts

```diff
@@ -23,13 +23,14 @@ export function getProductOptions({
     }
 
     for (let option of variant.options) {
-      if (option.group?.name) {
-        if (!typeOptions.hasOwnProperty(option.group.name)) {
-          typeOptions[option.group.name] = [];
+      const groupName = option.group?.translated?.name || option.group?.name;
```

Comment:
I think we've got a couple of places left in our code, where we go straight for the property instead of the translated.property. We should, in general, always go for the translated field and never use the original field of any entity.